src/System.Private.CoreLib/shared/System/MissingMemberException.cs

```diff
@@ -80,12 +59,20 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("MMSignature", Signature, typeof(byte[]));
         }
 
+        public override string Message
+        {
+            get
+            {
+                // do any desired fixups to classname here when ClassName not null.
+                return ClassName == null ? base.Message : SR.Format(SR.MissingMember_Name, ClassName + "." + MemberName + (Signature != null ? " " + FormatSignature(Signature) : string.Empty));
```

Comment:
The 200 characters long line is not an improvement.

Reply:
If you prefer I will switch it back to look like this. I think picked the one-liner style from corert rather than using existing style from coreclr

```
        public override String Message
         {
            get
            {
                if (ClassName == null)
                {
                    return base.Message;
                }
                else
                {
                    // do any desired fixups to classname here.
                    return SR.Format(SR.MissingMember_Name, ClassName + "." + MemberName + (Signature != null ? " " + FormatSignature(Signature) : ""));
                }
            }
        }
```

Reply:
That, or breaking the one-liner style to multiple lines. Either is fine with me - whatever you prefer.